composer.json

```diff
@@ -11,7 +11,11 @@
     }
   ],
   "require": {
-    "silverstripe/cms": ">=3.1"
+    "silverstripe/cms": ">=3.1",
+    "composer/installers": "*"
```

Comment:
This line isn't needed because framework requires it - also, your constraint is too liberal and should be `^1`